Browser_IDE/index.html

```diff
@@ -113,14 +249,18 @@ <h1>Let's write some code!</h1>
 				</div>
 			</div>
 			<div class="sk-collapsed-column sk-hidden">
-				<button type="button" title="Expand Program" onclick="collapseProgramViewToggle()"><i class="bi bi-terminal"></i></button>
+				<button type="button" title="Expand Program" onclick="collapseProgramViewToggle()"><i
+						class="bi bi-terminal"></i></button>
 				<button id="collapsedRunProgram" type="button" class="run-program" title="Build and Run">
 					<i class="bi bi-gear-fill"></i>
 					<i class="bi bi-play-fill"></i>
 				</button>
-				<button type="button" id="collapsedRestartProgram" title="Restart" style="display:none"><i class="bi bi-arrow-repeat"></i></button>
-				<button type="button" id="collapsedContinueProgram" title="Continue" style="display:none"><i class="bi bi-play-fill"></i></button>
-				<button type="button" id="collapsedStopProgram" title="Pause" style="display:none"><i class="bi bi-pause-fill"></i></button>
+				<button type="button" id="collapsedRestartProgram" title="Restart" style="display:none"><i
+						class="bi bi-arrow-repeat"></i></button>
+				<button type="button" id="collapsedContinueProgram" title="Continue" style="display:none"><i
+						class="bi bi-play-fill"></i></button>
+				<button type="button" id="collapsedStopProgram" title="Pause" style="display:none"><i
+						class="bi bi-pause-fill"></i></button>

```

Comment:
Better not to change code that isn't related to the PR - I agree we can improve the formatting, but for now let's leave these parts as-is :yum: Same comments for a few areas above.